distribution/Dockerfile-ubi

```diff
@@ -23,4 +23,6 @@ COPY ./internal/jobqueue/dbjobqueue/schemas /opt/migrate/schemas
 COPY --from=builder2 /opt/app-root/src/go/bin/tern /opt/migrate/
 
 EXPOSE 8080 8700
-ENTRYPOINT ["python3", "/opt/entrypoint.py", "--remote-worker-api", "--composer-api", "--composer-api-port", "8080"]
+ENV ENTRYPOINT_ADDITIONAL_ARGS ""
```

Comment:
I'm thinking of just removing this altogether and relying on the ability of k8s to override the entrypoint. https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/pod-v1/#entrypoint

Ideas? Feels cleaner to me.

Reply:
You mean removing `ENTRYPOINT_ADDITIONAL_ARGS` right? Meaning keep the default entrypoint of 
`ENTRYPOINT ["python3", "/opt/entrypoint.py", "--remote-worker-api", "--composer-api"]`